### PR TITLE
update composer dependencies to avoid problems on php 7.3

### PR DESCRIPTION
**Pull request by heddn**

### CODE REVIEW

**Comment by heddn** [comment]
This is a bit of a tough spot. To both support php 5.x and php 7.1+, we need to use different symfony dependencies. Any thoughts on how to get around this? This is currently stopping me from adopting php 7.3.

**Comment by heddn** [comment]
@greg-1-anderson any suggestions on this?

**Comment by greg-1-anderson** [comment]
I'm using Terminus with PHP 7.3, but then again, I installed with `git` and ran `composer update` with PHP 7.3.  Do the older versions of Symfony not work with PHP 7.3?

Terminus has:
```
    "platform": {
      "php": "5.5.38"
    }
```
That means that all of the dependencies for Terminus will work with PHP 5.5.38, and as I mentioned above, they also work with PHP 7.3.  Might that work here?

Note also, though, that Terminus will soon be delivered in a phar; that might simplify what is needed here.

**Comment by heddn** [comment]
It now works on PHP 7.3. And also on PHP 5.5.9.

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "minimum-stability": "beta",
     "prefer-stable": true,
     "require": {
-        "php": ">=5.5.9",
+        "php": "5.5.9  || ^7.0",
```
Comment (greg-1-anderson) [inline composer.json]: This excludes 5.6, and newer versions of 5.5. I think this line was fine the way it was; the config you added below should do the trick without this change.

Comment (heddn) [reply composer.json]: Yup, that fixed it still too. Manually tested on php 7.3. Do we need some automated tests there? I've modified travis to do that just now. Let's see how that fairs.

**Comment by heddn** [comment]
Travis doesn't seem to be testing much :(

**Comment by heddn** [comment]
Bump. I'm still blocked from updating to php 7.3 in my CI environment because I want to install terminus in it.

**Comment by heddn** [comment]
For now, using my own fork: https://github.com/heddn/terminus-installer/raw/php7.3-update/builds/installer.phar to get around this php 7.3 blocker.

**Comment by greg-1-anderson** [comment]
Note that Terminus 2 now builds a .phar on every release; you can just download that from GitHub.
```
wget https://github.com/pantheon-systems/terminus/releases/download/2.0.0/terminus.phar
chmod +x terminus.phar 
 ./terminus.phar self:update
```
From there, you can either rename and move to a directory in your $PATH, or maybe make a symlink etc.

Also note that there is a [docker image containing Terminus + plugins](https://github.com/pantheon-systems/docker-build-tools-ci) that you may use for automated testing on CI servers.

**Pull request merged**